packages/sdk-ts/src/client/indexer/grpc/IndexerGrpcSpotApi.ts

```diff
@@ -288,8 +288,9 @@ export class IndexerGrpcSpotApi extends BaseConsumer {
         SpotSubaccountTradesListResponse,
         typeof InjectiveSpotExchangeRPC.SubaccountTradesList
       >(request, InjectiveSpotExchangeRPC.SubaccountTradesList)
-
-      return IndexerGrpcSpotTransformer.tradesResponseToTrades(response)
+      return IndexerGrpcSpotTransformer.tradesResponseToTrades(
+        response as SpotTradesResponse,
```

Comment:
@bangjelkoski ^^ same comment as above

Reply:
We should try and resolve this without typecasting (same for above. Maybe the import in the `IndexerGrpcSpotTransformer` is from the `exchange-api` instead?

Reply:
@bangjelkoski this was resolved by merging in `master` branch, which updated the method call to `IndexerGrpcSpotTransformer.subaccountTradesListResponseToTradesList(response)`, so we're all good here. also was resolved for `IndexerGrpcDerivativeTransformer`